js/index.js

```diff
@@ -0,0 +1,102 @@
+function shuffle(array) {
+    let currentIndex = array.length,  randomIndex;
+
+    while (currentIndex !== 0) {
+
+        randomIndex = Math.floor(Math.random() * currentIndex);
+        currentIndex--;
+
+        [array[currentIndex], array[randomIndex]] = [
+            array[randomIndex], array[currentIndex]];
+    }
+
+    return array;
+}
+
+let expectedWord = []
+fetch('words.json')
+    .then(response => response.json())
+    .then(words => {
+        const wordsArr = words['fiveLetterWords']
+        const shuffledWords = shuffle(wordsArr)
+        expectedWord = shuffledWords[0].toUpperCase().split("")
+        console.log(expectedWord)
+        let attemptedWord = []
+        let characterCounter = 0
+// let attemptCounter = 0 - useful later
+
+// MAKE ON SCREEN LETTERS WORK
+        const onScreenLetters = document.querySelectorAll('.key')
+        const backspace = document.querySelector('.backspace')
+        const enter = document.querySelector('.enter')
```

Comment:
Should we rename these constants to e.g. `targetBackspace` to improve: 
- semantics and; 
- consistency (with `instructions.js`)?

Reply:
amended!